lib/openpgp_crypt_rsa.php

```diff
@@ -211,8 +213,13 @@ function decrypt($packet) {
   }
 
   static function try_decrypt_session($key, $edata) {
-    $key->setEncryptionMode(CRYPT_RSA_ENCRYPTION_PKCS1);
-    $data = @$key->decrypt($edata);
```

Comment:
phpseclib 2  was using a  "user notice"  , hence the `@` before to silence it , while phpseclib3 use exception , hence the try catch  to be backward compatible in term of interface

Reply:
however it may be better to let the exception raise , as having an exception is more explicit than returning null  (which does not tell what you've done wrong )

Reply:
I think in several places where we use this, we actually expect it to fail (hence the `@`) because we are trying a session key against a list of packets for example, and we skip where it fails.  So I think this approach is right for this particular procedure for now.